### DISCUSSION TITLE
Is there a way to Fire and forget using undici

### DISCUSSION THREAD

**Discussion by mandray** [nodejs/undici#839] [Q&A]
Is there a way you can send an http request and forget it. Meaning that the library under the hood won't wait for response. I know you can catch the response and just do nothing with it, but i'm trying to not even have the request come back to the javascript thread altogether. Similar to how UDP works

**Comment by mcollina** [comment]
This is not possible. You can avoid waiting for it yourself, e.g. `request(...).catch(() => {})`.
